sdk/nodejs/automation/localWorkspace.ts

```diff
@@ -840,8 +865,14 @@ export class LocalWorkspace implements Workspace {
         if (this.isRemote) {
             envs["PULUMI_EXPERIMENTAL"] = "true";
         }
+        // If pulumi points to a custom installation, run with
+        // IGNORE_AMBIENT_PLUGINS to ensure we pickup bundled bundled plugins
+        // from the installation instead of $PATH.
+        if (this.pulumi.command !== "pulumi") {
+            envs["IGNORE_AMBIENT_PLUGINS"] = "true";
```

Comment:
```suggestion
            envs["PULUMI_IGNORE_AMBIENT_PLUGINS"] = "true";
```

I think.

Makes me wonder though.  Would it be better to introduce a separate environment variable, something like `PULUMI_DEFAULT_TO_PULUMI_HOME` to default to looking up the plugins in `PULUMI_HOME`, but fall back to the `PATH`?  I don't know if there is any, but I imagine users with custom plugins could use them being in the PATH instead of in `PULUMI_HOME` to get them installed :thinking:

Reply:
If I understood correctly provider plugins are loaded from ~/.pulumi/plugins. The plugins that are loaded from path are the "bundled plugins" like `pulumi-language-nodejs`. There are some tests for providers that use $PATH to load things, but the new providers testing framework that is using automation API is not relying on this and is instead explicitly 

--- 

The alternative is to execute with PATH=$INSTALL_ROOT:$PATH 

To be honest I am not sure which is the better choice.

Reply:
Actually, I think prepending `$INSTALL_ROOT/bin` to `$PATH` is the better solution. That directory will only contain the CLI & bundled plugins and ensure we load the correct versions of these, but all other plugins will then use the normal lookup logic.

Reply:
Yeah I agree with that, prepending `$PATH` with `$INSTALL_ROOT` seems like the least surprising option here, especially given how pulumi loads plugins currently.